fbw-a32nx/src/systems/instruments/src/SD/Pages/Fuel/Fuel.tsx

```diff
@@ -354,3 +374,97 @@ const Pump = ({ x, y, onBus = 'DC_ESS', pumpNumber, centreTank, tankQuantity }:
         </g>
     );
 };
+
+interface CentreToInnerTransferProps {
+    x: number,
+    y: number,
+    side: 'L' | 'R',
+    /** centre tank level in gallons */
+    centreTankLevel: number,
+    modeSelectManual: boolean,
+    autoShutoffRequired: boolean,
+    onGround: boolean,
+}
+
+const CentreToInnerTransfer: FC<CentreToInnerTransferProps> = ({ side, centreTankLevel, x, y, modeSelectManual, autoShutoffRequired, onGround }) => {
+    // the valve is actually implemented as a pair of valves in series due to the way the MSFS fuel system triggers work
+    const [transferValveAuto] = useSimVar(`A:FUELSYSTEM VALVE OPEN:${side === 'R' ? 12 : 11}`, 'percent over 100', 1000);
+    const [transferValveInhibit] = useSimVar(`A:FUELSYSTEM VALVE OPEN:${side === 'R' ? 10 : 9}`, 'percent over 100', 1000);
+    const [transferValveSwitch] = useSimVar(`A:FUELSYSTEM VALVE SWITCH:${side === 'R' ? 10 : 9}`, 'boolean', 1000);
+    const [transferJunction] = useSimVar(`A:FUELSYSTEM JUNCTION SETTING:${side === 'R' ? 5 : 4}`, 'number', 1000);
+    const [transferValveFullyOpen, setTransferValveFullyOpen] = useState(false);
+    const [transferValveFullyClosed, setTransferValveFullyClosed] = useState(false);
+    const [centreTankLowLevel, setCentreTankLowLevel] = useState(false);
+
+    const [pumpClass, setPumpClass] = useState('GreenLine');
+    const [closedClass, setClosedClass] = useState('Hide');
+    const [openClass, setOpenClass] = useState('Hide');
+    const [faultClass, setFaultClass] = useState('Hide');
+    const [transferClass, setTransferClass] = useState('Hide');
+
```

Comment:
Are we sure those need to be stored in state. It seems like they can be computed from the simvars directly and don't need hooks.

Reply:
@tracernz or @Taz5150?

Reply:
Can’t say. I did not work on the Fuel.tsx file for this PR and do not have knowledge on that specific code. Sorry.